src/example4.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <dr/mhp.hpp>
+#include <fmt/core.h>
+
+namespace mhp = dr::mhp;
+using T = int;
+
```

Comment:
Please add a descriptive comment, eg.
/* The example presents operation of ... The result is stored in ... */

Reply:
I thought about describing all new examples together in the README file

Reply:
So a brief summary, like 
```/* add content of two 2-d arrays and display the results */```
The files have their names (not informing about the content, just exampleX), so as a reader of a tutorial, I would appreciate some additional textual sync between the description and the particular code.